stats/opencensus/e2e_test.go

```diff
@@ -237,6 +237,23 @@ func distributionDataLatencyCount(vi *viewInformation, countWant int64, wantTags
 	return nil
 }
 
+// waitForServerCompletedRPCs waits until both Unary and Streaming metric rows
+// appear for server completed RPC's view (by checking for length of rows to be
+// 2). Returns an error if both the Unary and Streaming metric not found within
+// the passed context's timeout.
+func waitForServerCompletedRPCs(ctx context.Context, fe *fakeExporter) error {
+	for ; ctx.Err() == nil; <-time.After(time.Millisecond) {
+		rows, err := view.RetrieveData("grpc.io/server/completed_rpcs")
+		if err != nil {
+			continue
+		}
+		if len(rows) == 2 {
```

Comment:
Instead of a len check here, should we implicitly check for 1 Unary and 1 Streaming RPC metric?

Reply:
Explicitly*. But sure.